app/controllers/payment-links/get-amount.controller.js

```diff
@@ -7,17 +7,25 @@ const lodash = require('lodash')
 const { response } = require('../../utils/response.js')
 const paths = require('../../paths')
 
-module.exports = (req, res) => {
-  const pageData = lodash.get(req, 'session.pageData.createPaymentLink', {})
-  const paymentLinkAmount = req.body['payment-amount'] || pageData.paymentLinkAmount || ''
-  const paymentAmountType = req.body['amount-type-group'] || pageData.paymentAmountType || ''
+module.exports = function showAmountPage (req, res, next) {
```

Comment:
Very minor but the guidance (you're potentially about to write 😆 ) might suggest doing 
```
function showAmountPage() {
}
module.exports = showAmountPage